docs/sources/mimir/visualize/native-histograms/_index.md

```diff
@@ -15,17 +15,24 @@ weight: 100
 
 # Visualize native histograms
 
-Prometheus native histograms is a data type in the Prometheus ecosystem that makes it possible to produce, store, and query a high-resolution [histogram](https://prometheus.io/docs/concepts/metric_types/#histogram) of observations.
+Native histograms are experimental and subject to change.
+
+Prometheus native histograms is a new data type beside the existing floating point time series in the Prometheus ecosystem that makes it possible to produce, store, and query a high-resolution [histogram](https://prometheus.io/docs/concepts/metric_types/#histogram) of observations.
 To learn more about the native histograms data type and how to start sending native histograms to Grafana Mimir,
 refer to [Send native histograms to Mimir]({{< relref "../../send/native-histograms" >}}).
 
+In general not all visualizations support displaying the new native histogram data type, however you can use the Prometheus Query Language (PromQL) to derive the usual floating point time series from native histograms that can be used in visualizations as before.
+
 ## Prometheus Query Language
 
-The Prometheus Query Language (PromQL) allows you to query native histogram metrics.
-PromQL queries of native histograms are different from those of classic histograms.
+The Prometheus Query Language (PromQL) allows you to query native histogram metrics. The data type of the result depends on the query and the underlying data.
 
 For more information about PromQL, refer to [Querying Prometheus](https://prometheus.io/docs/prometheus/latest/querying/basics/).
 
+The following sections describe common ways to derive floating point time series from native histograms data for visualizations and also how to convert existing queries using classic histograms into queries using native histograms.
```

Comment:
```suggestion
The following examples show common ways to derive floating point time series from native histograms data for visualizations and also how to convert existing queries using classic histograms into queries using native histograms.
```

Reply:
We try to avoid "documenting the documentation." Instead of `sections`, could we say `examples`, `queries`, or something more specific?